CONTRIBUTING.md

```diff
@@ -23,5 +23,5 @@ Kudos to you :balloon:
 
 
 
-Feel free to reach out to us on our [discord channel](https://discord.com/invite/jZQs6Wu). 
+Feel free to reach out to us on our [discord channel](https://discord.com/invite/jZQs6Wu). <br>
```

Comment:
`<br>` is achievable by having an extra line break. Not a big deal, we can still `accept` and `merge`